silverback/_cli.py

```diff
@@ -546,8 +547,18 @@ def create_payment_stream(
     sm = platform.get_stream_manager(network.chain_id)
     product = configuration.get_product_code(account.address, cluster.id)
 
+    accepted_tokens = platform.get_accepted_tokens(network.chain_id)
+    if token:
+        try:
+            token_symbol = Contract(token).symbol()
```

Comment:
I think you could do something like check if it's an address, and then do this without a try...except

Reply:
I've tried checking for an address without a `try... except` but even using `convert` will need one. The first method will check if it an address by association if the `Contract(token)` gives a value error but the new push checks using `convert` and will give a `ConversionError`. Both do the same thing but the second is more descriptive.

Reply:
see https://github.com/ApeWorX/silverback/pull/171#discussion_r1848416683